### ISSUE TITLE
Extra spaces in JSX

### ISSUE THREAD

**Issue by oliverjam** [fac20/Week10TT#5]
https://github.com/fac20/Week10TT/blob/91f8cace8f080b353c77aabcc3d19ade5586e104/src/App.js#L38-L41

You can probably delete any random extra spaces like this (the `{" "}`).

This is a weird thing that happens with Prettier that I thought I'd clarify. JSX differs slightly from HTML in how it handles whitespace. HTML is totally whitespace-sensitive (as in spaces in your code between inline elements will cause an actual space to appear on the page). JSX is whitespace-sensitive _between_ elements but not at the end of a line.

When this combines with Prettier aggressively putting things on new lines when the line gets too long you have a problem. E.g. if you have

```jsx
<span>Some stuff</span> <button>Other</button>
```
(with a space in the middle) if Prettier moved the button onto a new line that could potentially change your page layout (since the space would disappear). To ensure it doesn't break anything Prettier preserves the space by making it a JS string, using curly brackets and quotes. This forces React to keep it there.

tl;dr you can probs delete these unless you're using spaces to layout your HTML elements 🙃

**Comment by trishtracking** [comment]
thanks! yeh I couldn't figure that out this morning 😅
